build-scripts/get_files_including.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+import glob
+import sys
+
+
+def main(changed_files_file):
+    includes_to_sources = {}
+
+    # Load includes files list.
+    includes_files = glob.glob("obj/**/*.inc", recursive=True)
+    for includes_file_path in includes_files:
+        # substitute obj for src
+        # currently dont have to handle tidy
+        source_file = "src" + includes_file_path[3:-3] + "cpp"
+
+        with open(includes_file_path, 'r') as includes_file:
+            includes = includes_file.readlines()
+            for include in includes:
+                if include[0] != '.':
+                    # Out of actual output
+                    # gcc additionally prints an informational section like:
+                    # Multiple include guards may be useful for:
+                    break
+                # Format is one or more '.' characters, followed by a space,
+                # followed by the header
+                include_actual = include[include.index(' ') + 1:]
+                includes_to_sources.setdefault(
+                    include_actual,
+                    set(),
+                ).add(source_file)
+
+    changed_files = []
+    lintable_files = set()
+    with open(changed_files_file, 'r') as f:
+        changed_files = f.readlines()
+        for changed_file in changed_files:
+            for affected_file in includes_to_sources.get(changed_file, []):
+                lintable_files.add(affected_file)
+
+    for lintable_file in lintable_files:
+        print(lintable_file)
+
```

Comment:
Python LGTM.com check suggests adding a return value
```suggestion

    return 0

```